Gemfile

```diff
@@ -3,9 +3,12 @@ source 'https://rubygems.org'
 # 64bit Amazon Linux 2016.09 v2.3.0 running Ruby 2.3 (Puma)
 ruby '2.3.1'
 
-gem 'puma'
-
+gem 'activerecord'
+gem 'activerecord-postgis-adapter'
```

Comment:
True, it won't be a problem. `lib/active_record/connection_adapters/postgis/databases.rake` is file that defines rake tasks for activerecord-postgis-adapter. I just found out that we need to require this file and initialize PostGIS stuff.